### PR TITLE
Add logging out directions for auth0 lesson

### PR DESCRIPTION
**Pull request by ANURADHAJHA99**

### CODE REVIEW

File: oauth/o-auth.md [hunk 1]
```diff
@@ -61,29 +61,35 @@ The first thing you want to do is set up your credentials on google by following
 Please make sure that you secure ALL your private keys and sensitive information under your ``.env`` files! </br> 
 #### Step 2 - Connect Google credentials to Auth0 in Connections </br> </br> 
 Log into Auth0, on the left side you will see a menu with a ``Connections`` option. Select ``Connections`` then select the sub category named ``Social`` that will lead you to select the Gmail connection as follows: </br>
-<img src="https://github.com/mmedina3/Auth0-Lesson/blob/master/connectionstab.png"  width="500" height="500" /></br>
+<img src="https://github.com/mmedina3/Auth0-Lesson/blob/master/connectionstab.png" width="500" height="500" /></br>
 
 Once you click on ``Try it!`` you will see a modal pop up where you will have to input the Google credentials you created earlier. Just as follows: </br>
-<img src="https://github.com/mmedina3/Auth0-Lesson/blob/master/gmailconnectionmodal.png"  width="500" height="500" /></br></br>
+<img src="https://github.com/mmedina3/Auth0-Lesson/blob/master/gmailconnectionmodal.png" width="500" height="500" /></br></br>
```
Comment (ANURADHAJHA99) [inline oauth/o-auth.md]: The images are stored in the folder where other images are stored and I have asked the maintainer to merge my PR.
https://github.com/mmedina3/Auth0-Lesson - The following is the link to the image folder.

Comment (ANURADHAJHA99) [reply oauth/o-auth.md]: My PR consists of the images of the code snippets.

File: oauth/o-auth.md [hunk 2]
```diff
@@ -61,29 +61,35 @@ The first thing you want to do is set up your credentials on google by following
...
 #### Step 3 - Create Auth0 application to download code files </br> 
 Once you've successfully integrated your Google credentials with the connections section of Auth0 you can proceed with going to ``Applications`` and then type in ``React`` in the search bar. </br>
-<img src="https://github.com/mmedina3/Auth0-Lesson/blob/master/Screen%20Shot%202018-10-09%20at%205.58.09%20PM.png"  width="500" height="500" /></br></br>
+<img src="https://github.com/mmedina3/Auth0-Lesson/blob/master/Screen%20Shot%202018-10-09%20at%205.58.09%20PM.png" width="500" height="500" /></br></br>
+
+#### Step 4 - Create a working log-out button such that the button is shown when the user is authenticated, providing an option to log-out successfully </br>
+Once the user has successfully login,they are rendered with the option of log-out button, the following code in the *app.js* file can be added to obtain so.</br>
```
Comment (manaswinidas) [inline oauth/o-auth.md]: user has successfully 'logged in'

**Comment by manaswinidas** [comment]
Does this PR address any issues? Please reference the issue

File: oauth/o-auth.md (hunk 2 above)
Comment (ANURADHAJHA99) [reply oauth/o-auth.md]: made the following change in my recent commit.

**Comment by ANURADHAJHA99** [comment]
https://github.com/Techtonica/curriculum/issues/562  -  This is the issue the following PR is trying to solve

File: oauth/o-auth.md (hunk 1 above)
Comment (alodahl) [reply oauth/o-auth.md]: please move these from mmedina's repo to techtonica's

**Comment by ANURADHAJHA99** [comment]
I have closed the PR which was incorrect.

**Comment by ANURADHAJHA99** [comment]
As for the slides are concerned, can I get the write access to the slides?
I will surely edit the slides accordingly.

**Comment by ANURADHAJHA99** [comment]
I have added the files in the new commit.

**Comment by alodahl** [comment]
@ANURADHAJHA99 the slides should be editable for you. https://docs.google.com/presentation/d/18f-1EbjxLXyxuRoXNekpJ_o-pI1uiBzJxFw9ZoaNuVM/edit?usp=sharing

**Comment by ANURADHAJHA99** [comment]
I have edited the presentation.

**Comment by ANURADHAJHA99** [comment]
@alodahl I have edited the presentation.

**Comment by ANURADHAJHA99** [comment]
@alodahl I have done the changes that you asked for. Kindly review it.

**Comment by ANURADHAJHA99** [comment]
@alodahl I have removed the image files as you asked for.

**Pull request merged**